src/ucp/core/ucp_context.c

```diff
@@ -230,6 +230,10 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Maximal number of devices on which a RMA operation may be executed in parallel",
    ucs_offsetof(ucp_context_config_t, max_rma_lanes), UCS_CONFIG_TYPE_UINT},
 
+  {"EP_MAX_LANES", UCS_PP_MAKE_STRING(UCP_MAX_LANES),
+   "Maximal number of lanes per endpoint",
+   ucs_offsetof(ucp_context_config_t, ep_max_lanes), UCS_CONFIG_TYPE_UINT},
+
```

Comment:
imo, better to change just test

Reply:
IMO it's still a problem that we don't have a way to limit number of lanes on the endpoint, like we had in proto v1

Reply:
Yes, but in proto v1 we have separate variables for eager, rndv and RMA and this patch introduces a limit for all type of lanes. Will aliases work with it (if we need extra lane for KA or wireup)?